src/App.vue

```diff
@@ -48,12 +48,13 @@
       <MultipleHeadlines :count=4 />
     </CardGroup>
 
-    <CardSingle
+    <CardGroup
```

Comment:
Did this and added some styles so that when multiple items are pulled in, it removes the card footer and adds one per article instead.